frontend/js/restqueries.js

```diff
@@ -0,0 +1,82 @@
+try {
+	var staticURL = "http://ec2-34-208-19-115.us-west-2.compute.amazonaws.com:18080/character/";
+	var playerURL = "OtherTestChar";
+	var DnDObj = getUser();
+	//getJSONField("player");
+}
+catch (err) {
+	console.log(err);
+}
+//getJSONField("name");
+populateDNDFields();
```

Comment:
right now you're only pulling once, and changing it to not use a global variable won't change that. You just need to call populateDNDFields() in your try block.

You also need to change the following functions:

populateDNDFields() - > populateDNDFields(dbObj)
getJSONField(JSONFieldName) -> getJSONField(name,dbObj)

in your try block you call:

populateDNDField(DnDObj)

then in your populateDNDField(dbObj)

instead of calling getJSONFieldName(name), you call getJSONFieldName(name,dbObj)